src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -169,7 +182,7 @@ private void OnWorkspaceChanged(object? sender, WorkspaceChangeEventArgs args)
 
                     if (currentContextDocumentId != null && currentContextDocumentId.ProjectId == args.ProjectId)
                     {
-                        StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: 0, selectedItemUpdateDelay: 0, updateUIWhenDone: true);
+                        StartModelUpdateAndSelectedItemUpdateTasks(modelUpdateDelay: 0);
```

Comment:
all the calls to StartModelUpdateAndSelectedItemUpdateTasks passed `selectedItemUpdateDelay: 0`, so that parameter ogt removed.  All but one calls passed `updateUIWhenDone: true`.  The only one that passed `false` was when we used to present the items.  we'd quickly kick off hte task to select the item, then wait on it.  because we were doign this work on the UI, we didn't want it to also update the UI.

Now that i've gotten rid of that logic to kick off this task when we open teh nbavbar drowdown, there's no need for the updateUIWhenDone parameter either.